modules/microshift-uninstalling-lvms-csi-driver.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-driver_{context}"]
+= Uninstalling the CSI driver implementation
+
+To uninstall the installed CSI driver implementation, use the following procedure.
+
+.Prerequisites
+
+* {microshift-short} is installed and running.
+* You have deployed the CSI driver implementation.
+
+.Procedure
+
+. Delete the `vg-manager` by running the following command:
++
+[source,terminal]
+----
+$ oc delete -n openshift-storage daemonset.apps/vg-manager
+----
++
+.Example output
+[source,terminal]
+----
+daemonset.apps "vg-manager" deleted
+----
+. Delete the `lvms-operator` by running the following command:
++
+[source,terminal]
+----
+$ oc delete -n openshift-storage deployment.apps/lvms-operator
+----
++
+.Example output
+[source,terminal]
+----
+deployment.apps "lvms-operator" deleted
+----
+. Delete the `topolvm-provisioner` by running the following command:
```

Comment:
```suggestion
. Delete the `topolvm-provisioner` StorageClass by running the following command:
```
not sure what the appropriate way of writing the Kind ("StorageClass"?), so you may want to 2x check that.

Reply:
done